.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
please remove this file. thanks.

Reply:
This is the debugger configuration for Visual Studio Code. Instead of removing I would suggest to add configurations for other IDE's like PHP Storm too to get an out of the box running setup...

Reply:
If you decide to do so we should also think about to eventually exclude this files for export in .gitattributes...